10/debian-10/rootfs/opt/bitnami/scripts/postgresql-env.sh

```
@@ -199,6 +202,9 @@ export POSTGRESQL_DAEMON_GROUP="postgres"
 
 # PostgreSQL settings
 export POSTGRESQL_INIT_MAX_TIMEOUT="${POSTGRESQL_INIT_MAX_TIMEOUT:-60}"
+export PGCTLTIMEOUT="${PGCTLTIMEOUT:-60}"
```

Instead of setting this environment variable here, as it is used by `pg_ctl`, I think it would be better to set it in the `pg_ctl` line the same way `PGDATA` is set

---

Hm, in that case I'd rather go for using the explicit invocation parameter (`--timeout=<seconds>`) instead of relying on an environment variable. Same actually applies to `PGDATA` (parameter would be `--pgdata=<datadir>`). As long as there is an explicit parameter available, that should be preferred over relying on environment variables, which are picked up or not picked up by a command (in contrast to a parameter, which is either supported or not and would throw an error at you, not silently ignore it).

---

Hi,
 Do you mind renaming this environment variable as `POSTGRESQL_PGCTLTIMEOUT` ?, this way we have a unified interface in the container as all the "parameters" are `POSTGRESQL_xxxx`.